ipn/ipnserver/server.go

```diff
@@ -303,8 +302,8 @@ func (s *Server) checkConnIdentityLocked(ci *ipnauth.ConnIdentity) error {
 			return inUseOtherUserError{fmt.Errorf("Tailscale already in use by %s, pid %d", active.User().Username, active.Pid())}
 		}
 	}
-	if su := s.serverModeUser; su != nil && ci.UserID() != su.Uid {
-		return inUseOtherUserError{fmt.Errorf("Tailscale already in use by %s", su.Username)}
+	if cu := s.b.CurrentUser(); cu != "" && cu != ci.UserID() {
+		return inUseOtherUserError{fmt.Errorf("Tailscale already in use by %s", s.b.CurrentUser())}
```

Comment:
A userID here in the error message sucks a lot more than a username. Add a TODO to fix that up? Should probably rename the LocalBackend methods to be CurrentUserID and SetCurrentUserID too.

Reply:
oops missed this, sending a follow up